### PR TITLE
Add Early Starkers

### PR DESCRIPTION
**Pull request by Hikmo**

### CODE REVIEW

**Comment by graemeblackwood** [comment]
@Hikmo Currently your banner crops the text and stars on the right hand side.

 I have prepared a Figma preview for you that shows you the safe area for content in the banner. Anything outside the area should be expected to crop: https://www.figma.com/file/wDHepez6fPmXXuF3SEfks0/Early-Starkers-Dappland-content-template?node-id=0%3A1

File: next.config.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ const nextConfig = {
       "www.frenslands.xyz",
       "raw.githubusercontent.com",
       "lh3.googleusercontent.com",
+      "i.ibb.co"
```
Comment (janek26) [inline next.config.js:30]: please move sources inside the repo, see other dapp's json

Comment (Hikmo) [reply next.config.js:30]: maybe I didn't get it correct, this is where I have added the outer sources into next config file
https://github.com/argentlabs/dappland/pull/18/commits/c4528a8e12168d7a1d12bd2ab532c33eab74478e

Comment (janek26) [reply next.config.js:30]: right, but that's not the case anymore
Please rebase your branch and see others, like here:
https://github.com/argentlabs/dappland/blob/develop/data/aspect.json#L40

Static files are now inside the repo

**Comment by graemeblackwood** [comment]
Implemented by https://github.com/argentlabs/dappland/pull/78

**Pull request closed without merging**